### DISCUSSION TITLE
How to add Bluedart Shipping  ?

### DISCUSSION THREAD

**Discussion by chahal26** [bagisto/bagisto#6722] [Q&A]
Hi I am new to Bagisto. Any idea, how can I add Bluedart shipping?

**Comment by amit-webkul** [comment]
Hi @chahal26,

If you want to add the Bluedart shipping method in Bagisto, first you need to check the Blue Dart Shipping method requirements. Make sure its requirements are compatible with Bagisto. Once this is verified, then we can make Blue Dart shipping method work, by making it compatible from code end.

**Note :-  For making any changes in the code I would suggest you to make a custom package for Blue Dart Shipping Method.**

For your Reference ( Blue Dart Developer Guide) :- https://support.postmen.com/en/article/blue-dart-developer-guide-api-credentials-lyjdjo/
